extensions/civetweb/processors/ListenHTTP.cpp

```diff
@@ -40,12 +40,12 @@ core::Property ListenHTTP::SSLCertificate("SSL Certificate", "File containing PE
 core::Property ListenHTTP::SSLCertificateAuthority("SSL Certificate Authority", "File containing trusted PEM-formatted certificates", "");
 
 core::Property ListenHTTP::SSLVerifyPeer(
-    core::PropertyBuilder::createProperty("SSL Verify Peer")->withDescription("Whether or not to verify the client's certificate (yes/no)")->isRequired(false)->withAllowedValue<std::string>("yes")
-        ->withAllowedValue("no")->withDefaultValue("no")->build());
+    core::PropertyBuilder::createProperty("SSL Verify Peer")->withDescription("Whether or not to verify the client's certificate (yes/no)")->isRequired(false)->withAllowableValue<std::string>("yes")
+        ->withAllowableValue("no")->withDefaultValue("no")->build());
 
 core::Property ListenHTTP::SSLMinimumVersion(
     core::PropertyBuilder::createProperty("SSL Minimum Version")->withDescription("Minimum TLS/SSL version allowed (SSL2, SSL3, TLS1.0, TLS1.1, TLS1.2)")->isRequired(false)
-        ->withAllowedValue<std::string>("SSL2")->withAllowedValue("SSL3")->withAllowedValue("TLS1.0")->withAllowedValue("TLS1.1")->withAllowedValue("TLS1.2")->withDefaultValue("SSL2")->build());
+        ->withAllowableValue<std::string>("SSL2")->withAllowableValue("SSL3")->withAllowableValue("TLS1.0")->withAllowableValue("TLS1.1")->withAllowableValue("TLS1.2")->withDefaultValue("SSL2")->build());
```

Comment:
This could be changed to use the new version with set param.

Reply:
I left it there as an example that both ways are allowed.